Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -148,6 +160,73 @@ function createStackEntry(props: any): any {
  * ### Constants
  *
  * `currentHeight` (Android only) The height of the status bar.
```

Comment:
Sounds reasonable, but it'll probably be a long one since it's used in a lot of things and it's not a simple transition because a lot of people do the really bad thing of hardcoding the `currentHeight` value in a constant, which means it's a tougher transition to using something non-constant.

Reply:
What is the preferred way to deprecate a property in RN?

Reply:
I don't remember. 😄 Search for `deprecate` in .js files.
Okay, let's deprecate it in another PR.